pontoon/static/js/widgets/errors.css

```diff
@@ -0,0 +1,12 @@
+
+ul.errors {
+    padding: .3em;
+    margin: 0;
+    list-style: none;
+    background: #ccc;
+}
+
+ul.errors li.error {
+    color: #F36;
+    font-size: 1.1em;
+}
```

Comment:
What do I need to do in the UI to test this CSS?

Reply:
its kinda hard to trigger in normal operation - one way is to open 2 admin windows showing same resources. In first window select all and link/unlink - then do the same in second window with same resources - that should throw an error

Reply:
I get a weird error message if I do this:
![screen shot 2018-03-21 at 15 45 12](https://user-images.githubusercontent.com/626716/37716457-e266c6b0-2d1e-11e8-8fab-c4b67fa8d784.png)

Reply:
let me check that again

Reply:
@mathjazz good spot - fixed

Reply:
Thanks for the update! I'd still change the message from:
```
data: Unable to link the resources requested
```
to
```
Unable to link the requested resources
```

Reply:
its in the context of it being a django form - so its the `data` field that is wrong - but we can make it so that it adds a form error, rather than a field error

Reply:
As long as we drop the field name (and change word order), I'm fine. It doesn't really tell much to the end user in this case.